.circleci/config.yml

```diff
@@ -8,11 +8,11 @@ checkout_project_root: &checkout_project_root
     path: ~/openlineage
 
 
-# Only trigger CI job on release (=X.Y.Z)
+# Only trigger CI job on release (=X.Y.Z) with possible (rcX)
 only-on-release: &only-on-release
   filters:
     tags:
-      only: /^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?$/
+      only: /^[0-9]+(\.[0-9]+){2}(rc[0-9]+)?$/
```

Comment:
I don't think this regex would work for jvm release, for example: `0.1.0-rc1` (which is more common). I think we'll want to support `X.Y.Z-rcN` for jvm specific releases and `X.Y.ZrcN` for python (as per [`PEP 440`](https://www.python.org/dev/peps/pep-0440/#pre-release-spelling)). You can update the regex to support both by adding `-?`:

```
^[0-9]+(\.[0-9]+){2}(-?rc[0-9]+)?$
```

Reply:
Makes sense - but maybe have two separate release paths for clients? We don't necessarily want to release both clients together.

Reply:
@wslulciuc for now let's go with `-?` version

Reply:
>My line of thinking is that, when we update the OpenLineage spec, it would generate the client models and trigger a release with each client since they are within the same repo.

That's one case where we want to update all the clients. Second one is when we want to fix bug in particular client implementation, where release of other clients is not needed, and I'm not sure how to reconcile those two.